en/developer_guide/block_override.rst

```diff
@@ -0,0 +1,105 @@
+Block Override
+===================
+
+Overloading the template of a block
+===================================
+
+Overloading the template of a block is achieved by creating a template in the following folder structure :
+
+.. code-block::
+
+    |_ app
+        |_ Resources
+            |_ {BUNDLE_NAME}
+                |_ views
+                    |_ {PATH_TEMPLATE}
+                        |_ {FILENAME}
+
+
+Example :
+---------
+
+You need override the display template of block "Content List Block" in Backoffice to display the datas
+that you consider important.
+
+.. code-block::
+
+    |_ Open-orchestra
+        |_ open-orchestra-cms-bundle
+            |_ BackofficeBundle
+                |_ Ressources
+                    |_ views
+                        |_ Block
+                            |_ ContentList
+                                |_ show.html.twig
+
+``Open-orchestra/open-orchestra-cms-bundle/BackofficeBundle/Ressources/views/Block/
+ContentList/show.html.twig``
+
+You simply create your template in next folder :
+
+.. code-block::
+
+    |_ app
+        |_ Resources
+            |_ OpenOrchestraBackofficeBundle
+                |_ views
+                    |_ Block
+                        |_ ContentList
+                            |_ show.html.twig
+
+``app/Ressources/OpenOrchestraBackofficeBundle/views/Block/ContentList/show.html.twig``
+
+See again `symfony2 documentation`_
+
+Overloading directly a block
+=========================
+
+Before acceding this part, he is important reading the documentation `block parameter`_.
```

Comment:
Before accessing this part, it's important to understand the `block parameter`_. documentation.